File: angular6/src/app/shared/models/projectWorker.ts
import { User } from "./user";
import { HourForDepartment } from "./hourForDepartment";
import { PresentDay } from "./pressentDay";
import { Project } from "./project";

export class ProjectWorker {
    projectId: number;
    userId:number;
    projectName: string;
    hoursForProject: number;
    project: Project;
    userWithoutPassword:User;
    sumHoursDone:number;



}

File: webAPI-tasks-not/BOL/Models/ProjectWorker.cs
using BOL.HelpModel;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BOL.Models
{
   public class ProjectWorker
    {
        public int ProjectId { get; set; }
        public int UserId { get; set; }
        public decimal HoursForProject { get; set; }
        public Project Project { get; set; }
        public UserWithoutPassword User { get; set; }
        public decimal sumHoursDone { get; set; }
    }
}
